lte/gateway/configs/kernsnoopd.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+log_level: INFO
+
+# collect_interval is the interval in seconds at which kernsnoopd collects
+# counters from the kernel
+collect_interval: 10
+
+# ebpf_programs are the ebpf source filenames that kernsnoopd will load.
+ebpf_programs:
+  - packet_count
```

Comment:
List of eBPF programs to load. `packet_count` is the current one. I will also provide one which listens to TCP state change events so performance impact can be lowered. #8173 suggests another program to keep count of open file descriptors.